assets/constants.ts

```diff
@@ -74,6 +74,12 @@ export default {
     "History", "Mathematics", "Medicine", "Music", "Physics", "Programming",
     "Reading", "Statistics"],
 
+  // These classrooms are shown in the add learner group syllabus filters.
+  "SEARCH_DROPDOWN_CLASSROOMS": ["math"],
```

Comment:
Why this filter? shouldn’t all classrooms be shown?

Reply:
Added comment mentioning to update this filter in the config domain file